packages/serverpod/lib/src/database/database_pool_manager.dart

```diff
@@ -30,15 +41,16 @@ class DatabasePoolManager {
   DatabasePoolManager(
     SerializationManagerServer serializationManager,
     this.config,
-  ) {
+  ) : _poolSettings = pg.PoolSettings(
+          maxConnectionCount: 10,
```

Comment:
Question: This is not relevant for this PR but is this a configuration we should expose for our users? It might be nice to be able to adjust this.

Reply:
I agree with this! :)